monitoring/src/monitors/bootstrap_monitor.rs

```diff
@@ -37,6 +37,19 @@ impl BootstrapMonitor {
         }
     }
 
+    #[inline]
+    pub fn level(&self) -> usize {
+        self.level
+    }
+    #[inline]
+    pub fn set_level(&mut self, level: usize)  {
+        // only set peer current head level for values higher than the current level
+        self.level = if self.level > level { self.level } else { level }
+    }
+    #[inline]
+    pub fn set_downloaded_blocks(&mut self, downloaded_blocks: usize)  {
```

Comment:
What is intended difference between `set_level` and this method? both are mutating the same variable `self.level`.

Reply:
set_downloaded_blocks is more for initialization and special cases, where you don't intend to do any checks